app/domain/authentication/constraints/any_constraint.rb

```diff
@@ -0,0 +1,18 @@
+module Authentication
+  module Constraints
+
+    # This constraint validates that at least one of the initialized restrictions exist.
+    # So, any one of them must be found in the given restrictions when calling validate(given_restrictions).
+    class AnyConstraint
+
+      def initialize(any:)
```

Comment:
@abrahamko what does this class expect `any` to be? is a string? maybe a hash? if so - a hash of what?

i think that a comment is in place

Reply:
Comment added on top of class